### DISCUSSION TITLE
Solis Inverter & Solis Cloud Integration

### DISCUSSION THREAD

**Discussion by K3str3l** [slipx06/sunsynk-power-flow-card#351] [Solis]
Hi All,

Does anyone have a working example of this card using a Solis Inverter & the Solis Cloud Integration? I've spent several hours trying to configure it - but can't seem to find a lot of the entities - particular around battery and power consumption.

Any direction appreciated!
thanks in advance

**Comment by Roving-Ronin** [comment]
@K3str3l  just to confirm, you have looked at the example provided for Solis users?

https://slipx06.github.io/sunsynk-power-flow-card/examples/solis.html

BTW if like other inverter vendors and their Cloud systems, you want to NOT use that method, as they typically are only updating data to you about every 15min. The preferred method is via local (modbus) access, that is updated every ~30 seconds.

**Comment by Pho3niX90** [comment]
The example above should give what you need. Which integration are you currently using?

Solis cloud luckily is faster than most, but still, it only updates every 5minutes, not fast enough for most automations.

For Solis Modbus, I wrote the following to accomplish it https://github.com/Pho3niX90/solis_modbus/ , docs https://solis-modbus.readthedocs.io/en/latest/

**Reply by K3str3l** [reply]
thanks for replying @Roving-Ronin and @Pho3niX90 . Yes, I have looked at the Solis example linked - but I just don't have all those entitiles available.
I'm using the Solis Cloud integration - https://github.com/hultenvp/solis-sensor in conjunction with myenergi - https://github.com/CJNE/ha-myenergi
